ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -34,6 +34,7 @@ module('Acceptance | allocation detail', function (hooks) {
   hooks.beforeEach(async function () {
     server.create('agent');
 
+    server.create('node-pool');
```

Comment:
This has made me desperately want a `factory.beforeCreate()` in mirage, but it looks like this isn't a thing. Thanks for putting all the work in for this!

Reply:
It would make things so much easier 😿